params/network_upgrades_test.go

```diff
@@ -177,88 +205,99 @@ func TestVerifyNetworkUpgrades(t *testing.T) {
 		name          string
 		upgrades      *NetworkUpgrades
 		avagoUpgrades upgrade.Config
-		expected      bool
+		valid         bool
 	}{
 		{
-			name: "ValidNetworkUpgrades for latest network",
+			name: "ValidNetworkUpgrades_for_latest_network",
 			upgrades: &NetworkUpgrades{
 				SubnetEVMTimestamp: utils.NewUint64(0),
 				DurangoTimestamp:   utils.NewUint64(1607144400),
 				EtnaTimestamp:      utils.NewUint64(1607144400),
 			},
 			avagoUpgrades: upgradetest.GetConfig(upgradetest.Latest),
-			expected:      true,
+			valid:         true,
 		},
 		{
-			name: "Invalid Durango nil upgrade",
+			name: "Invalid_Durango_nil_upgrade",
 			upgrades: &NetworkUpgrades{
 				SubnetEVMTimestamp: utils.NewUint64(1),
 				DurangoTimestamp:   nil,
 			},
-			avagoUpgrades: upgrade.GetConfig(constants.MainnetID),
-			expected:      false,
+			avagoUpgrades: upgrade.Mainnet,
+			valid:         false,
 		},
 		{
-			name: "Invalid Subnet-EVM non-zero",
+			name: "Invalid_Subnet-EVM_non-zero",
 			upgrades: &NetworkUpgrades{
 				SubnetEVMTimestamp: utils.NewUint64(1),
 				DurangoTimestamp:   utils.NewUint64(2),
 			},
-			avagoUpgrades: upgrade.GetConfig(constants.MainnetID),
-			expected:      false,
+			avagoUpgrades: upgrade.Mainnet,
+			valid:         false,
 		},
 		{
-			name: "Invalid Durango before default upgrade",
+			name: "Invalid_Durango_before_default_upgrade",
 			upgrades: &NetworkUpgrades{
 				SubnetEVMTimestamp: utils.NewUint64(0),
 				DurangoTimestamp:   utils.NewUint64(1),
 			},
-			avagoUpgrades: upgrade.GetConfig(constants.MainnetID),
-			expected:      false,
+			avagoUpgrades: upgrade.Mainnet,
+			valid:         false,
 		},
 		{
-			name: "Invalid Mainnet Durango reconfigured to Fuji",
+			name: "Invalid_Mainnet_Durango_reconfigured_to_Fuji",
 			upgrades: &NetworkUpgrades{
 				SubnetEVMTimestamp: utils.NewUint64(0),
 				DurangoTimestamp:   utils.TimeToNewUint64(upgrade.GetConfig(constants.FujiID).DurangoTime),
 			},
-			avagoUpgrades: upgrade.GetConfig(constants.MainnetID),
-			expected:      false,
+			avagoUpgrades: upgrade.Mainnet,
+			valid:         false,
 		},
 		{
-			name: "Valid Fuji Durango reconfigured to Mainnet",
+			name: "Valid_Fuji_Durango_reconfigured_to_Mainnet",
 			upgrades: &NetworkUpgrades{
 				SubnetEVMTimestamp: utils.NewUint64(0),
 				DurangoTimestamp:   utils.TimeToNewUint64(upgrade.GetConfig(constants.MainnetID).DurangoTime),
 			},
-			avagoUpgrades: upgrade.GetConfig(constants.FujiID),
-			expected:      false,
+			avagoUpgrades: upgrade.Fuji,
+			valid:         false,
 		},
 		{
-			name: "Invalid Etna nil",
+			name: "Invalid_Etna_nil",
```

Comment:
nit (no action to take) That's a nice addition to use underscores, although a bit out of scope; we can keep it now that it's done, but I think we should address this and similar issues at repo-wide scale after the repo-merge so it's done everywhere, since this likely applies to a lot of test files.